### DISCUSSION TITLE
Octave-compatible syntax for Matlab developers

### DISCUSSION THREAD

**Discussion by lfarv** [atcollab/at#210] [Octave AT]
A few remarks for Matlab developers to avoid compatibility problems:

- Do not use the recent (R2016b)  string type. Octave treats double-quoted strings (Ex.: "Hello") as `character vectors` (like 'Hello') instead of `strings`,

- Do not use `missing`, `ismissing`, `fillmissing` introduced in R2017b.

**Comment by matsievskiysv** [comment]
Here's a [good resource](https://wiki.octave.org/Differences_between_Octave_and_Matlab) on this matter.
